production/helm/loki/values.yaml

```diff
@@ -739,6 +776,20 @@ read:
     targetCPUUtilizationPercentage: 60
     # -- Target memory utilisation percentage for the read
     targetMemoryUtilizationPercentage:
+    # -- Behavior policies while scaling.
+    behavior: {}
```

Comment:
again with behavior, how would you feel about taking a more opinionated stance with good defaults and the ability to override?

Reply:
TBH, while the default configuration for the write component is conservative to avoid frequent scale-down events, it's difficult to establish optimal default settings for other components, as their ideal scaling behavior depends on the specific environment and workload.

Reply:
yeah, fair point, agreed. thanks for the feedback!